Resources/ServerInfo/Guidebook/Antagonist/Traitors.xml

```diff
@@ -0,0 +1,33 @@
+<Document>
+  # Traitors
+
+  - A Traitor is an antagonist employed by the Syndicate. They have access to various Syndicate tools and weapons through their [color=#a4885c]Uplink[/color].
+  - Traitors can use codewords to identify themselves to one another in order to cooperate.
+  - In a Traitor round, any player can be a Traitor aside from department heads, the captain, or any security officers.
+  - Traitors should focus primarily on completing their objectives successfully by the end of the round.
+
+  ## Uplink
+
+  - The most important tool as a Traitor for purchasing tools and weapons with [color=#a4885c]telecrystals[/color] (TC)
```

Comment:
This line should say "uplink" instead of using the section header. Also, maybe state how traitors get their telecrystals (which is, obviously, that they start out with them)
```suggestion
  - The uplink is the most important tool as a Traitor, as it can purchase tools and weapons with [color=#a4885c]telecrystals[/color] (TC)
```

Reply:
Suggested change was made and telecrystals were given more info. Kept the header since all relevant info below it is regarding the Uplink. Check below for the new guidebook.